src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.ComposeCache.targets

```diff
@@ -41,7 +41,6 @@ Copyright (c) .NET Foundation. All rights reserved.
       <PreserveCacheLayout Condition="'$(PreserveCacheLayout)' == ''">true</PreserveCacheLayout>
       <SkipOptimization Condition="'$(RuntimeIdentifier)' == ''">true</SkipOptimization>
       <_TFM Condition="'$(_TFM)' == ''">$(TargetFramework)</_TFM>
-      <FX_Version Condition="'$(FX_Version)' == ''">$(_ShortFrameworkVersion)</FX_Version>
```

Comment:
the user is supposed to set this, if not then SDK will deduce a default through RuntimeFrameworkVersion, which is what i am making it move to now.
previously was disabling the implicit reference and adding an explicit ref, but changing it as per @eerhardt 's   https://github.com/dotnet/sdk/pull/761#discussion_r98922463